rllib/env/meta_env.py

```diff
@@ -0,0 +1,33 @@
+from gym.core import Env
+
+
+class MetaEnv(Env):
+    """
+    Wrapper around OpenAI gym environments and meta-learning environments
+    """
+
+    def sample_tasks(self, n_tasks):
+        """
```

Comment:
```suggestion
        """Samples task of the meta-environment
```

Please fix the first line to be like this.